grid2op/multi_agent/utils.py

```diff
@@ -74,4 +78,77 @@ def __eq__(self, other):
             and self._current_agent == other._current_agent
             and self.selected_agent == other.selected_agent
         )
-        
\ No newline at end of file
+
+
+class ClusterUtils:
+    """
+    Utility class for clustering substations
+    """
+    
+    # Create connectivity matrix
+    @staticmethod
+    def create_connectivity_matrix(env):
+        """
+        Creates a connectivity matrix for the given grid environment.
+
+        The connectivity matrix is a 2D NumPy array where the element at position (i, j) is 1 if there is a direct 
+        connection between substation i and substation j, and 0 otherwise. The diagonal elements are set to 1 to indicate 
+        self-connections.
+
+        Args:
+            env (grid2op.Environment): The grid environment for which the connectivity matrix is to be created.
+
+        Returns:
+            np.ndarray(env.n_sub, env.n_sub): A 2D NumPy array representing the connectivity matrix of the grid environment.
```

Comment:
Returns:
      - connectivity_matrix: A 2D Numpy array of dimension (env.n_sub, env.n_sub) representing the substation connectivity of the grid environment.

Reply:
Updated the docstring